html/elements/script.json

```diff
@@ -865,6 +933,39 @@
                 }
               }
             },
+            "source_document": {
+              "__compat": {
+                "description": "<code>source</code> key <code>document</code> value",
+                "support": {
+                  "chrome": {
+                    "version_added": "121"
+                  },
+                  "chrome_android": "mirror",
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "ie": {
+                    "version_added": false
+                  },
+                  "oculus": "mirror",
+                  "opera": "mirror",
+                  "opera_android": "mirror",
+                  "safari": {
+                    "version_added": false
+                  },
+                  "safari_ios": "mirror",
+                  "samsunginternet_android": "mirror",
+                  "webview_android": "mirror"
+                },
+                "status": {
+                  "experimental": true,
+                  "standard_track": true,
+                  "deprecated": false
+                }
+              }
+            },
```

Comment:
@chrisdavidmills we crossed our streams again :-)

I think we can remove this bit as I added this as a `url` and `where` parameters in #22188. We're making `source: url` and `source: document` optional (and maybe eventually removing it?) as it can be implied from whether `url` and `where` exists. Therefore don't think we should add that.